client/src/pages/Opportunities/PostOpportunity/PostOpportunity.js

```diff
@@ -5,6 +5,7 @@ import Form from '../../../objects/Form/Form.js';
 import Alert from '../../..//components/Alert/Alert.js';
```

Comment:
You can add validation in this file. However, there are disadvantages to doing it this way.

1. DRY (Don't repeat yourself)
If I want to use a zipcode validation in my PutOpportunity.js form, or elsewhere, I'd have to create a new test. To allow us to re-use the test zip code, you should put in somewhere it will be re-usable at.

Read my other comments to see how forms work below.